Build/openssl.props

```diff
@@ -1,35 +1,60 @@
 <?xml version="1.0" encoding="utf-8"?>
-<Project ToolsVersion="4.0" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
-  <PropertyGroup Label="PlatformType">
-    <platformType Condition="'$(ApplicationType)' == 'Android'">Android</platformType>
-    <platformType Condition="'$(ApplicationType)' == ''">Win32</platformType>
-  </PropertyGroup>  
+<Project
+    ToolsVersion="4.0" xmlns="http://schemas.microsoft.com/developer/msbuild/2003"
+    InitialTargets="SSLBuildDebug"
+>
+<!--Project ToolsVersion="4.0" xmlns="http://schemas.microsoft.com/developer/msbuild/2003"-->
   <PropertyGroup>
-    <HCSettingsFile>$([MSBuild]::GetDirectoryNameOfFileAbove($(MSBuildThisFileDirectory), hc_settings.props))\hc_settings.props</HCSettingsFile>
+    <HCBuildRoot Condition="'$(HCBuildRoot)'==''">$([MSBuild]::GetDirectoryNameOfFileAbove($(MSBuildThisFileDirectory), build.root))\</HCBuildRoot>
+    <libcryptoName>libcrypto.$(HCLibPlatformType)</libcryptoName>
+    <libsslName>libssl.$(HCLibPlatformType)</libsslName>
+    <opensslSourceRoot>$(HCBuildRoot)External\openssl\</opensslSourceRoot>
+    <opensslGeneratedHeaders>$(HCBuildRoot)External\generatedHeaders\$(HCLibPlatformType)\</opensslGeneratedHeaders>
+    <doref>lol nope</doref>
```

Comment:
Cant you just leave this undefinded?

Reply:
Ugh, testing code that shouldn't have been checked in